src/opcuabridge/signed.h

```diff
@@ -0,0 +1,49 @@
+#ifndef OPCUABRIDGE_SIGNED_H_
+#define OPCUABRIDGE_SIGNED_H_
+
+#include "common.h"
+
+namespace opcuabridge {
+class Signed {
+ public:
+  Signed() {}
+  virtual ~Signed() {}
+ public:
+  const int& get_numberOfTokens() const { return numberOfTokens_; }
+  void set_numberOfTokens(const int& numberOfTokens) { numberOfTokens_ = numberOfTokens; }
+  const std::vector<int>& get_tokens() const { return tokens_; }
+  void set_tokens(const std::vector<int>& tokens) { tokens_ = tokens; }
+  const int& get_timestamp() const { return timestamp_; }
+  void set_timestamp(const int& timestamp) { timestamp_ = timestamp; }
+ protected:
+  int numberOfTokens_;
+  std::vector<int> tokens_;
```

Comment:
Is the numberOfTokens_ duplicating information that is already in tokens_ ?

Reply:
Interface conforms to uptane spec. And it may be still useful when hashes implementation get changed.